### ISSUE TITLE
Thank you for your excellent work

### ISSUE THREAD

**Issue by xmhh** [imperial-qore/TranAD#55]
Thank you for your excellent work. When I read your articles and code, I have the following questions:
1. Training details. Except for TranAD, you don’t seem to use batch size in the training of the model, in other word, the batch size of the training is all 1, and the gpu is not used. Will this setting affect the backpropagation of the gradient? And can the speed continue to be optimized?
3. The GDN you reproduced does not use embedding layer, but it can reach or even exceed the results of the original model on SWaT?

Hope to get your help, thank you very much!

**Comment by shreshthtuli** [comment]
Thanks for your interest in this work.
To train the baselines, please use the original code repositories (links mentioned in the paper).
In our paper, we used the GDN with the Series2Graph approach (patented algorithm, not reproduced in this public repo). You can modify GDN as per https://helios2.mi.parisdescartes.fr/~themisp/series2graph/ to get the model used for generating the results in the paper. The improvement compared to the original code is due to the exhaustive hyperparameter search (as used in TranAD, for fair comparison) in our experiments.
